cmd/updater/versionCmd.go

```diff
@@ -36,12 +37,14 @@ func init() {
 	versionCmd.AddCommand(checkCmd)
 	versionCmd.AddCommand(getCmd)
 
-	checkCmd.Flags().StringVarP(&versionBucket, "bucket", "b", "", "S3 bucket containing updates.")
 	checkCmd.Flags().BoolVarP(&semanticOutput, "semantic", "s", false, "Human readable semantic version output.")
+	checkCmd.Flags().StringVarP(&packageName, "package", "p", "node", "Get version of specific package.")
+	checkCmd.Flags().StringVarP(&versionBucket, "bucket", "b", "", "S3 bucket containing updates.")
 
 	getCmd.Flags().StringVarP(&destFile, "outputFile", "o", "", "Path for downloaded file (required).")
-	getCmd.Flags().StringVarP(&versionBucket, "bucket", "b", "", "S3 bucket containing updates.")
 	getCmd.Flags().Uint64VarP(&specificVersion, "version", "v", 0, "Specific version to download.")
+	getCmd.Flags().StringVarP(&packageName, "package", "p", "node", "Get version of specific package.")
```

Comment:
replace the "node" string with a constant - this is not just a "claim" for a node package name; it's the naming conventions we use for creating package names. It should be a const string. Ideally, we would derive the same string when the package is created ( but we could do that on a separate, future, PR )